src/ui/layout/RecentObjectsList.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+<div
+    class="c-tree-and-search l-shell__tree"
+>
+    <ul
+        class="c-tree-and-search__tree c-tree c-tree__scrollable"
+    >
+        <recent-objects-list-item
+            v-for="(recentObject) in recentObjects"
+            :key="recentObject.navigationPath"
+            :object-path="recentObject.objectPath"
+            :navigation-path="recentObject.navigationPath"
+            :domain-object="recentObject.domainObject"
+            @openAndScrollTo="openAndScrollTo($event)"
+        />
+    </ul>
+</div>
+</template>
+
+<script>
+const MAX_RECENT_ITEMS = 20;
+const LOCAL_STORAGE_KEY__RECENT_OBJECTS = 'mct-recent-objects';
+import RecentObjectsListItem from './RecentObjectsListItem.vue';
+export default {
+    name: 'RecentObjectsList',
+    components: {
+        RecentObjectsListItem
+    },
+    inject: ['openmct'],
+    props: {
+
+    },
+    data() {
+        return {
+            recents: []
+        };
+    },
+    computed: {
+        recentObjects() {
+            return this.recents.filter((recentObject) => {
+                return recentObject.location !== null;
```

Comment:
are we filtering out deleted objects here?

Reply:
yes, but i'm not sure we need this since we listen to composition changes now and RecentObjects are populated with mutables..

Reply:
@scottbell So I messed around with this a bit, and actually this is handling the case of something being in someone's recent items, it being deleted in persistence (perhaps while someone else was using the app), and then loading up OpenMCT.

If we get rid of this filter, then recent objects would populate with "Missing" items if they were deleted in persistence while the user was away. Is this something we would want? cc: @charlesh88